app/controllers/errors_controller.rb

```diff
@@ -38,15 +50,45 @@ def notify_subscribers
   end
 
   def resolve
-    if !@error.resolved_at.nil?
-      @error.update_attributes(resolved_at: nil)
+    #has values when we resolve issues through the error sidebar
+    errors_per_page = params[:per_page] || 5
+    page = params[:page]
+    if params[:error_ids]
+      ids = params[:error_ids]
+      resolved = to_boolean(params[:resolved]) if params[:resolved]
+      resolve_issues(ids, resolved, errors_per_page, page, @error)
+    elsif params[:individual_resolve]
+      ids = [@error.id]
+      resolved = !@error.resolved_at.nil?
+      resolve_issues(ids, resolved, errors_per_page, page, @error)
     else
-      @error.update_attributes(resolved_at: DateTime.now)
+      raise 'Could not find error!'
     end
   end
 
   private
 
+  def resolve_issues (ids, resolved = true, errors_per_page, page, current_error)
+    errors = current_error.website.grouped_issues.order('last_seen DESC')
+    if resolved
+      GroupedIssue.where(id: ids).update_all(resolved_at: nil, status: 3) # 3 = unresolved
+      resolved = errors.where('resolved_at IS NOT NULL')
+      @sidebar = resolved.page(page).per(ids.size).offset(errors_per_page)
+      @pagination = resolved.page(page).per(errors_per_page).offset(ids.size)
```

Comment:
It was because I was using 2 variables to hold different types of errors. But ur right...if I put them in the same variable I won't have to repeat the last 2 lines.